src/userstyles.yml

```diff
@@ -269,6 +269,13 @@ userstyles:
       app-link: "https://github.com/libreddit/libreddit"
       current-maintainers: [*isabelroses]
       past-maintainers: [*andreasgrafen]
+  linkedin:
+    name: LinkedIn
+    category: social
+    icon: LinkedIn
+    readme:
+      app-link: "https://www.linkedin.com"
+      current-maintainers: [*isabelroses]
```

Comment:
Please add @sgoudham to the current maintainers

Reply:
Please **do not**